Zend/tests/short_closures/minimal_capture_007.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+Short closures minimal capture: foreach
+--FILE--
+<?php
+
+$a = 1;
+$x = 1;
+$k = 1;
+$v = 1;
+
+$f = fn () {
+    $ret = [];
+    foreach (range(0, $a) as $k => $v) {
+        $ret[] = [$k, $v];
+    }
+    return $ret;
```

Comment:
It would be good to know if using `$k` or `$v` after the loop would result in a capture, as the `foreach` could not run which would not assign those variables.

Reply:
This is tested in Zend/tests/short_closures/minimal_capture_002.phpt (using `for` and not `foreach`, though). I will add a variation :+1: